mycroft/skills/__main__.py

```diff
@@ -77,6 +77,19 @@ def _starting_up():
     check_connection()
 
 
+def try_update_system(platform):
+    bus.emit(Message('system.update'))
+    msg = Message('system.update', {
+        'paired': is_paired(),
+        'platform': platform
+    })
+    resp = bus.wait_for_response(msg, 'system.update.processing')
+
+    if resp.data.get('processing', True):
```

Comment:
Should a check be made here if `resp` is None since a timeout will return None?

Reply:
Nice catch! Updated.

Reply:
Yeah something like that will be needed, otherwise unpaired desktops will hang on start